### PR TITLE
feat: docker compose support and meta/bin/bootstrap script

### PR DESCRIPTION
**Pull request by chuckadams**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,6 @@
 /old_code
 /build
 .env
+
+node_modules/
```
Comment (sarah-savage) [inline .gitignore:7]: Probably don't need this?

File: compose.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+services:
+  aspiresync:
+    # image: aspirepress/apiresync:latest
```
Comment (sarah-savage) [inline compose.yml:3]: No need for this to be here commented out?

File: compose.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      target: devbuild
+
```
Comment (sarah-savage) [inline compose.yml:9]: Generally I prefer to keep whitespace out of the service defs but that's personal preference and not something I'm going to hold the PR over.

File: docker/init-db.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
```
Comment (sarah-savage) [inline docker/init-db.sh:1]: I'm curious what the advantage is over using the ENV variables of POSTGRES_USER and POSTGRES_DB.

**Review by sarah-savage — APPROVED** [review]
Just some notes, but overall this is fine.

File: docker/init-db.sh (hunk 4 above)
Comment (chuckadams) [reply docker/init-db.sh:1]: It is using those envars, which both default to `postgres`.

**Pull request merged**